deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -0,0 +1,114 @@
+id: hawkular-cassandra-node-dynamic-pv
+kind: Template
+apiVersion: v1
+name: 'Hawkular Cassandra Node Template: Persistent Storage'
```

Comment:
The use of id threw me off a bit as well, I've always used auto-generated ids, and set metadata.name for kube/openshift objects. I left it there just because I figured there was a reason for it.